test/cpu_reduce_util.hpp

```diff
@@ -71,22 +71,25 @@ static inline bool float_equal_one(T x)
     return false;
 };
 
+// use combination of "<=" and ">=" for float values equality checking to remove DeepCode warning
+// caused by direct using of "=="
```

Comment:
In this specific case, DeepCode warning is a misleading garbage that doesn't worth commenting (except disabling that check).

We need to reach consensus on this topic, so let's discuss.